network/NetworkGen/classes/Can.py

```diff
@@ -195,6 +196,17 @@ def __init__(
 
         self._check_val_roles()
 
+    def crc32(self):
+        crc = crc32(self.name.encode())
+        crc = crc32(self.start_bit.to_bytes(4, 'little'), crc)
+        crc = crc32(self.native_representation.bit_width.to_bytes(4, 'little'), crc)
+        crc = crc32(self.native_representation.signedness.value.encode(), crc)
+        crc = crc32(self.native_representation.signedness.value.encode(), crc)
+        crc = crc32(self.native_representation.endianness.value.to_bytes(4, 'little'), crc)
+        crc = crc32(str(self.scale).encode(), crc)
+        crc = crc32(str(self.offset).encode(), crc)
```

Comment:
The only thing g i can think that were missing here is units, but unclear. For exmaple someone changes torque request from N to mN

Reply:
added